src/lib/jsonParseV2/parseBatchQuery.cpp

```diff
@@ -60,15 +60,15 @@ std::string parseBatchQuery(ConnectionInfo* ciP, BatchQuery* bqrP)
 
   if (!document.IsObject())
   {
-    alarmMgr.badInput(clientIp, "JSON Parse Error");
+    alarmMgr.badInput(clientIp, "JSON Parse Error", "");
```

Comment:
I do not know if the 2nd description can lead to a misunderstandings

```suggestion
    alarmMgr.badInput(clientIp, "JSON Parse Error", "JSON Object not found");
```

Reply:
Ok